index.js

```diff
@@ -6,11 +6,13 @@ See the accompanying LICENSE file for terms.
 
 'use strict';
 
-var isRegExp = require('util').isRegExp;
+var isRegExp    = require('util').isRegExp;
+var randomBytes = require('randombytes');
```

Comment:
I used that so as not to break the serializer when run in a browser. If the serializer intended to be node-only I'll just use `crypto.randomBytes()`